tests/loaders/test_mqtt.py

```diff
@@ -61,20 +63,21 @@ def mock_client(mock_paho: MagicMock) -> Iterator[MagicMock]:
 
 
 @pytest.mark.parametrize(
-    "alias, host, disks, disk, seek, read, expected",
+    "alias, hosts, disks, disk, seek, read, expected",
     [
         ("host1", "host1", [3], 0, 0, 3, b"\x00\x01\x02"),  # basic
         ("host2", "host2", [10], 0, 1, 3, b"\x01\x02\x03"),  # + use offset
         ("group1", "host3", [10], 0, 1, 3, b"\x01\x02\x03"),  # + use alias
         ("group2", "host4", [10, 10, 1], 1, 1, 3, b"\x01\x02\x03"),  # + use disk 2
+        ("group3", ["host4", "host5"], [10, 10, 1], 1, 1, 3, b"\x01\x02\x03"),  # + use disk 2
     ],
 )
 @patch.object(time, "sleep")  # improve speed during test, no need to wait for peers
 def test_remote_loader_stream(
     time: MagicMock,
     mock_client: MagicMock,
     alias: str,
-    host: str,
+    hosts: list[str],
```

Comment:
The other `hosts` parameters are still just strings?

Reply:
Good find, funny that the test still worked, but it makes sense.